src/py21cmfast/wrapper.py

```diff
@@ -1637,6 +1637,7 @@ def ionize_box(
             previous_ionize_box,
             spin_temp,
             pt_halos,
+            init_boxes,  # JBM: added this
```

Comment:
can take out the comments that you added it... we can `git blame` it to get this info :-)